ts/features/payments/checkout/hooks/usePaymentFailureSupportModal.tsx

```diff
@@ -82,8 +92,25 @@ const usePaymentFailureSupportModal = ({
     "";
 
   const zendeskAssistanceLogAndStart = () => {
+    if (!isReady(zendeskPaymentCategory)) {
+      return;
+    }
+    const { payments } = zendeskPaymentCategory.value;
+    const subCategory = getSubCategoryFromFaultCode(payments, faultCodeDetail);
+    // attach the main zendesk category to the ticket
+    addTicketCustomField(
+      zendeskCategoryId,
+      defaultZendeskPaymentCategory.value
+    );
+
+    if (subCategory) {
+      // if a subcategory is found, we attach its id and value to the ticket
+      const { value, zendeskSubCategoryId } = subCategory;
+      addTicketCustomField(zendeskSubCategoryId, value);
+      return;
```

Comment:
The `return` statement here is unnecessary, as it interrupts the ticket opening process. As a result, tapping on the CTA to contact support from the displayed bottom sheet has no effect.

Reply:
My bad, I missed that earlier. Solved here https://github.com/pagopa/io-app/pull/6522/commits/773571b83473d5269cba4d523ab685224fb73327